packages/renderer/index.d.ts

```diff
@@ -503,7 +465,7 @@ declare module '@react-pdf/renderer' {
           orientation: Orientation;
         },
       ) => Style;
-      flatten: (...style: (Style[] | Style | undefined)[]) => Style;
+      flatten: (...style: Style[] | Style | undefined[]) => Style;
```

Comment:
Rest parameters should always Array
Seems it should be `...style: (Style[] | Style | undefined)[]`